src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -356,6 +356,8 @@ def __init__(self, model_proxy=None, name=None):
 
     @classmethod
     def _native_name(cls):
+        if USE_CPP:
+            return "activity_classifier"
         return None
```

Comment:
Just curious what happens when returning None?

Reply:
Good question! So the class doesn't get registered (via `RegistrationMetaClass`) and placed in `MODEL_NAME_MAP`. We can use this to our advantage by returning `None` from `_native_name` when we don't want that class to be registered. This way we control if the Python or C++ class is registered upon importing turicreate. This fixes the bug of wrapping a C++ model with the Python class wrapper.

Reply:
Great!!Thanks!